src/test/java/bio/terra/service/snapshotbuilder/utils/CriteriaQueryBuilderTest.java

```diff
@@ -248,21 +248,20 @@ void generateRollupCountsQueryForCriteriaGroupsList(SqlRenderContext context) {
     // FIXME: is query correct? It doesn't contain the concept IDs 11 and 10.
```

Comment:
I think this comment is old and was removed in another PR

Reply:
That's true, I haven't yet merged `develop` back into this branch to pick up that change